cincinnati/src/plugins/internal/channel_filter.rs

```diff
@@ -104,6 +108,10 @@ impl InternalPlugin for ChannelFilterPlugin {
                 graph,
                 parameters: internal_io.parameters,
             })
+        })
+        .map_err(move |e| {
+            error!("error filtering graph: {}", e);
```

Comment:
Policy-engine should be already logging this error, can you confirm?

Reply:
Right, this is being ignored - removed